tutorials/get-started-with-dqlite-in-go.md

```diff
@@ -0,0 +1,23 @@
+This step-by-step guide walks you through the setup of an Dqlite example application. 
+By the end of this tutorial you will have a fully-functional Dqlite cluster deployed on your machine and you'll be able to interact with it through a HTTP client.
+
+## What you’ll need:
+
+* A working station, e.g. a laptop
```

Comment:
"working station" is not correct I think

Reply:
Fixed